content/docs/command-reference/repro.md

```diff
@@ -268,14 +269,13 @@ $ dvc repro --downstream
 Data and pipelines are up to date.
 ```
 
-The reason being that the `text.txt` file is a dependency in the last stage of
-the pipeline (used by default by `dvc repro`), This last `count` stage is
-dependent on `filter` stage, which happens first in this pipeline (shown in the
-following figure):
+The reason being that the `text.txt` file is not a dependency in the last stage
+of the pipeline, used as the default target by `dvc repro`. `text.txt` is a
+dependency of the `filter` stage, which happens earlier (shown in the figure
+below), so it's skipped given the `--downstream` option.
```

Comment:
I think there is a mistake here as `filter` stage would not be skipped but it will be executed and hence the complete pipeline with an output -

![image](https://user-images.githubusercontent.com/46163196/88225625-6e843c80-cc88-11ea-943b-7168ea8ff09e.png)

Reply:
The whole example has to be rewritten as mentioned in https://github.com/iterative/dvc.org/pull/1615#pullrequestreview-453636252